pkg/providers/version/suite_test.go

```diff
@@ -53,6 +66,10 @@ var _ = BeforeSuite(func() {
 	ctx, stop = context.WithCancel(ctx)
 	awsEnv = test.NewEnvironment(ctx, env)
 
+	serverVersion, err := env.KubernetesInterface.Discovery().ServerVersion()
+	Expect(err).ToNot(HaveOccurred())
+	k8sVersion = fmt.Sprintf("%s.%s", serverVersion.Major, strings.TrimSuffix(serverVersion.Minor, "+"))
+
```

Comment:
We have a helper for this! I'd just call this inline rather than assigning to a global.

https://github.com/aws/karpenter-provider-aws/blob/bbb499628ade784af2511d300c8ad3e15587fdca/test/pkg/environment/aws/expectations.go#L361-L365